.github/workflows/cron.yaml

```diff
@@ -0,0 +1,59 @@
+name: Update VEX Hub
+on:
+  schedule:
+    - cron: "0 0 * * *" # Every day
+  workflow_dispatch:
+
+permissions:
+  contents: write
+  actions: read
+
+jobs:
+  update:
+    name: Update VEX Hub
+    runs-on: ubuntu-latest
```

Comment:
nit:
I understand that we don't use any third party utilities (except git), but I think it's better to pin the Ubuntu version to avoid unexpected errors after upgrading the Ubuntu version in Runner (like we got the error with oras in trivy-db and trivy-java-db)